packaging/samba.spec.j2

```diff
@@ -575,10 +575,12 @@ Requires: %{name} = %{samba_depver}
 Requires: %{name}-client-libs = %{samba_depver}
 Requires: %{name}-common-libs = %{samba_depver}
 Requires: %{name}-libs = %{samba_depver}
+%if 0
 Requires: python3-talloc
 Requires: python3-tevent
 Requires: python3-tdb
 Requires: python3-ldb
```

Comment:
Ok...these are currently part of _samba-client-libs_ and I could see a `Requires:` for it under _python3-samba_. So either:
* we avoid these `Requires:` for _python3-*_ of internal libraries from _python3-samba_
OR
* we add `Provides:` for _python3-*_ of internal libraries under _samba-client-libs_

Reply:
Yes we could add provides.
But instead I avoided the `Requires:` ... the `%if 0` is essentially commenting this out, no?

Reply:
> Yes we could add provides.
> But instead I avoided the `Requires:` ... the `%if 0` is essentially commenting this out, no?

Agreed.
How about adding a comment along with `%if 0` for better understanding?

Reply:
Yeah, I could do that. Could also do that in the other several instances.. ;-)

Reply:
Thinking about it, since we do ship these python3-foobar libs in official locations, I though it's more appropriate to add the `Provides: ` lines to the `samba-client-libs` package instead. Let me know what you think, @anoopcs9

Reply:
> Thinking about it, since we do ship these python3-foobar libs in official locations, I though it's more appropriate to add the `Provides: ` lines to the `samba-client-libs` package instead. Let me know what you think, @anoopcs9

Fine by me.